book/notes/liquidation-math.md

```diff
@@ -0,0 +1,108 @@
+# Margin Account Liquidation Math
+
+This is a short note to explain the math behind margin account liquidation. Dango's margin account liquidation is based on Euler's dynamic liquidation bonus method, or "Dutch Liquidation". Please see this [paper](https://docs.euler.finance/Dutch_Liquidation_Analysis.pdf) for more details.
+
+## Defintions
+
+* Collateral power: The adjustment factor for the collateral value of a given collateral token. Meaning, if the collateral token has a collateral power of 0.9, then the value of the collateral token is 90% of its actual value.
+* Utilization rate: The ratio of the total value of the debts to the total value of the account's collateral adjusted for the collateral power. When the utilization rate is greater than 1, the account is undercollateralized and can be liquidated.
+* Health factor: The inverse of the utilization rate. When the health factor is less than 1, the account is undercollateralized and can be liquidated.
+* Liquidation bonus: The amount of extra collateral that is awarded to the liquidator to incentivize them to liquidate the account.
+
+## Calculating The Maximum Repayable Debt
+
+When liquidating the account, we set a target utilization rate (which is a fraction between 0 and 1) that we want to reach after the liquidation. We then calculate the maximum amount of debt that can be repaid to reach this target utilization rate.
```

Comment:
How is this rate a governance decided constant or is it determined in some other ways?

Reply:
Yeah I added it in the AppConfig. Can update the doc to be clear about that.